cmd/installer/kotscli/kotscli.go

```diff
@@ -79,8 +84,16 @@ func Install(opts InstallOptions) error {
 		installArgs = append(installArgs, "--airgap-bundle", opts.AirgapBundle)
 		maskfn = MaskKotsOutputForAirgap()
 	}
+
 	if opts.ConfigValuesFile != "" {
 		installArgs = append(installArgs, "--config-values", opts.ConfigValuesFile)
+	} else if len(opts.ConfigValues) > 0 {
+		configValuesFile, err := createConfigValuesFile(opts.ConfigValues, opts.RuntimeConfig.EmbeddedClusterHomeDirectory())
```

Comment:
let's store this file in the embedded cluster tmp dir and defer remove it as it can contain sensitive info